common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -315,6 +315,9 @@ def student_view(self, context):
         child_context = {} if not context else copy(context)
 
         for child in self._get_selected_child_blocks():
+            if child is None:
+                logger.error('Skipping display for child block that is None')
+                continue
```

Comment:
To avoid this eventually becoming one of those noisy errors that we forget why we logged and then blindly rip out two years later to reduce logging noise...

* can you add a comment explaining why this case happens, and why we're handling it like this?
* iff there's a "better" fix for this we know of or think we should find, can you link to a ticket for that?
* can you add any potentially-helpful info to the message? (ik that user ID and URL are already logged, so it may be fine as-is)

Reply:
@kdmccormick 

>     * can you add a comment explaining why this case happens, and why we're handling it like this?

Done, at least as far as we know at this point...

>     * iff there's a "better" fix for this we know of or think we should find, can you reference a ticket # in the comment?

I've linked to underlying TNL bug ticket w/ a `TODO` note.

>     * can you add any potentially-helpful info to the message? (ik that user ID and URL are already logged, so it may be fine as-is)

The only requests for data made by team members include these fields, so I think we're good for now. I'll happily add extra context, should we find a need.